src/viztracer/modules/quicktime.c

```diff
@@ -142,12 +142,23 @@ quicktime_init()
         start_ns[i] = get_system_ns();
         int64_t start_after = get_system_ts();
         start_ts[i] = start_before + (start_after - start_before) / 2;
-        t0_ts += start_ts[i];
-        t0_ns += start_ns[i];
     }
 
-    t0_ts /= CALIBRATE_SIZE;
-    t0_ns /= CALIBRATE_SIZE;
+    // Do the expensive average calculation outside the measurement loop
+    int64_t ts_remainder = 0, ns_remainder = 0;
```

Comment:
Looks good. Could you split this into two declarations? I'm always trying to avoid declaration like this - it's often okay for a few non-pointer variables, but it gets ugly when it's a pointer or the variable name is too long or there are too many of them. Just as a habit.

Reply:
Sure! Pushed it now